.secrets.baseline

```diff
@@ -0,0 +1,85 @@
+{
+  "exclude": {
+    "files": "^go.sum|^.secrets.baseline$",
+    "lines": null
+  },
+  "generated_at": "2020-04-13T20:45:35Z",
+  "plugins_used": [
+    {
+      "name": "AWSKeyDetector"
+    },
+    {
+      "name": "ArtifactoryDetector"
+    },
+    {
+      "base64_limit": 4.5,
+      "name": "Base64HighEntropyString"
+    },
+    {
+      "name": "BasicAuthDetector"
+    },
+    {
+      "name": "CloudantDetector"
+    },
+    {
+      "hex_limit": 3,
+      "name": "HexHighEntropyString"
+    },
+    {
+      "name": "IbmCloudIamDetector"
+    },
+    {
+      "name": "IbmCosHmacDetector"
+    },
+    {
+      "name": "JwtTokenDetector"
+    },
+    {
+      "keyword_exclude": null,
+      "name": "KeywordDetector"
+    },
+    {
+      "name": "MailchimpDetector"
+    },
+    {
+      "name": "PrivateKeyDetector"
+    },
+    {
+      "name": "SlackDetector"
+    },
+    {
+      "name": "SoftlayerDetector"
+    },
+    {
+      "name": "StripeDetector"
+    },
+    {
+      "name": "TwilioKeyDetector"
+    }
+  ],
+  "results": {
+    "config.yaml": [
+      {
+        "hashed_secret": "2f6f0c24ab741f0e2a0ffdd200adca35f2f72f87",
```

Comment:
what are these?

Reply:
I set up https://github.com/Yelp/detect-secrets in this repo. This is the baseline detection result

Reply:
These "hashed_secret" listed here are not really secrets. They are either hashed high entropy strings (SHAs) or non-sensitive secrets like `FLYTE_AWS_SECRET_ACCESS_KEY: miniostorage`